guide/english/c/if/index.md

```diff
@@ -60,5 +60,5 @@ int main () {
 ## Output
 ```
 -> a is not less than 5!
--> Value of a is : 100
```

Comment:
The value of a is initialized as 10 above. It cannot be 100.

Reply:
Yes, it should be 10